src/WSGI.py

```diff
@@ -1,6 +0,0 @@
-import sys
-from printapp import app
-
-if __name__ == "__main__":
-    app.run(debug=True, port=5001)
```

Comment:
We still need this file. As the readme says, it's how we run the application (both locally and in production).